platform/ios/Integration Tests/MGLTestLocationManager.m

```diff
@@ -0,0 +1,51 @@
+#import "MGLTestLocationManager.h"
+
+// Used to supply integration tests with a simulated location manager.
+// Methods that are empty are not used within integration tests and are
+// therefore unimplemented.
+
+@implementation MGLTestLocationManager
+
+@synthesize delegate;
+
+- (instancetype)init
+{
+    if (self = [super init]) {
+    }
+    return self;
+}
+
+- (CLAuthorizationStatus)authorizationStatus { return [CLLocationManager authorizationStatus]; }
```

Comment:
Since this is a test manager, how about returning `kCLAuthorizationStatusAuthorized` here?